src/resources/account/google/google.controller.js

```diff
@@ -0,0 +1,45 @@
+const userService = require('resources/user/user.service');
+const authService = require('auth.service');
+const googleService = require('resources/account/google/google.service.js');
+
+const securityUtil = require('security.util');
+const config = require('config');
+
+const createUserAccount = async (userData) => {
+  const signupToken = await securityUtil.generateSecureToken();
+
+  const user = await userService.create({
+    firstName: userData.given_name,
+    lastName: userData.family_name,
+    email: userData.email,
+    isEmailVerified: true,
+    signupToken,
```

Comment:
@Mar1nka as we create account with isEmailVerified = true, signupToken is not needed so we can remove it from here